README.md

```diff
@@ -120,6 +120,22 @@ fastify.setErrorHandler(function (err, req, reply) {
 
 ## Options
 
+### `utf8` <Boolean> (optional, default: true)
+
+User-ids or passwords containing characters outside the US-ASCII
+character repertoire will cause interoperability issues, unless both
+communication partners agree on what character encoding scheme is to
+be used. If utf8 is set to true the server will send the 'charset' parameter
+to indicate a preference of "UTF-8", increasing the probability that
+clients will switch to that encoding.
```

Comment:
```suggestion
User-ids or passwords containing characters outside the US-ASCII
character set will cause interoperability issues, unless both
communication partners agree on what character encoding scheme is to
be used. If utf8 is set to true the server will send the 'charset' parameter
to indicate a preference of "UTF-8", increasing the probability that
clients will switch to that encoding.
```

Reply:
I took that sentence from the RFC :). But i like your suggestion better.

Reply:
The person who wrote the RFC needs to crack open a dictionary https://www.merriam-webster.com/dictionary/repertoire